libcontainer/seccomp/patchbpf/enosys_unsupported.go

```diff
@@ -1,18 +1,3 @@
 // +build !linux !cgo !seccomp
 
 package patchbpf
```

Comment:
perhaps file can be removed altogether (trying to avoid an empty package for these build tags)

Reply:
Maybe add a comment saying "we are keeping this file to avoid empty package errors"?